Code/Framework/AzCore/Tests/Math/Vector2Tests.cpp

```diff
@@ -148,14 +149,14 @@ namespace UnitTest
 
     TEST(MATH_Vector2, TestGetNormalized)
     {
-        EXPECT_THAT(AZ::Vector2(3.0f, 4.0f).GetNormalized(), IsClose(AZ::Vector2(3.0f / 5.0f, 4.0f / 5.0f)));
-        EXPECT_THAT(AZ::Vector2(3.0f, 4.0f).GetNormalizedEstimate(), IsClose(AZ::Vector2(3.0f / 5.0f, 4.0f / 5.0f)));
+        EXPECT_THAT(AZ::Vector2(3.0f, 4.0f).GetNormalized(), IsCloseTolerance(AZ::Vector2(3.0f / 5.0f, 4.0f / 5.0f), Constants::Tolerance));
```

Comment:
nitpick - I think IsCloseTolerance(x, Constants::Tolerance) is no different from IsClose.

Reply:
In this case is different because it'd be using `UnitTest::Constants::Tolerance` instead of `AZ::Constants::Tolerance`, but I can see that it could cause confusion (specially since `using namespace AZ;` has been added at the top of these math files, which is not preferred).

To make it explicit I'll rename `UnitTest::Constants::Tolerance` variables to `UnitTest::Constants::SimdTolerance`

Reply:
As we discussed offline, I had confused it with the value from the AZ namespace. +1 on the rename to make it less easy to mix up.